test/machineslib.py

```diff
@@ -235,6 +235,19 @@ def run_admin(self, cmd, connectionName='system'):
         else:
             return m.execute(cmd)
 
+    def deleteDisk(self, target, vm_name="subVmTest1"):
```

Comment:
You just defined this function without using this anywhere. Forgot to commit something?